python/pip_install/tools/wheel_installer/wheel_installer.py

```diff
@@ -289,6 +298,17 @@ def _generate_build_file_contents(
     )
 
 
+def _generate_bin_bzl_contents(entrypoints: dict[str, str]) -> str:
+    return textwrap.dedent(
+        """\
+        # Autogenerated by wheel_installer.py
```

Comment:
```suggestion
        # Autogenerated by wheel_installer.py#_generate_bin_bzl_contents
```

Thanks for adding the "generated by" text. Those comments have been very helpful as I've tried to trace code around. I just a small suggestion to point directly to the part doing it. I'm also +1 on including any other extra info in the comment that makes it easier for maintainers.